rfc/2019-12-11-Talent-Plan-Courses.md

```diff
@@ -0,0 +1,31 @@
+# Proposal: Talent Plan Courses
+
+- Author(s): [Ling Jin](https://github.com/JinLingChristoher)
+- Last updated: 2019-12-11
+- Discussion at: [Incubating Program: Talent Plan Courses](https://github.com/pingcap/community/issues/130)
+
+There are plenty of people interested in TiDB but lacking background knowledge, so we propose to create a series of courses to help those people enter the field infrastructure systems and TiDB ecosystem. Course participants should have developed good knowledge and skills in system programming after finishing course materials.
+
+## Course design
+
+![course map](./course-ex.png)
```

Comment:
1. the course graph needs to be updated.
2. please put `course-ex.png` to the `media` folder. you can name it to `rfc-talent-plan-courses.png`